image/store.go

```diff
@@ -67,21 +67,21 @@ func NewImageStore(fs StoreBackend, lss LayerGetReleaser) (Store, error) {
 
 func (is *store) restore() error {
 	err := is.fs.Walk(func(dgst digest.Digest) error {
-		img, err := is.Get(IDFromDigest(dgst))
+		img, err := is.Get(ID(dgst))
```

Comment:
Instead of repeating the .WithField() I tend  to first define partially filled log for the scope that the information is relevant
So, in this case, here I'd add:
```suggestion
		img, err := is.Get(IDFromDigest(dgst))
		log := logrus.WithField("digest", dgst)
```

Reply:
Ah, yes, we do that in various places. It came up  when I was writing it, but not sure why I didn't do that.

Generally, I try to avoid using `log` for these (as it's easy to confuse for "this is the `log` package), so we use a slightly different name (e.g. `logger`)